src/components/Project/ProjectHeader/GnosisSafeBadge.tsx

```diff
@@ -21,17 +32,90 @@ export function GnosisSafeBadge({ href }: { href: string }) {
     )
   }
 
-  return (
-    <Tooltip
-      placement="bottom"
-      title={
-        <Trans>
-          This project is owned by a Safe.{' '}
-          <Link href={href}>See transactions</Link>.
-        </Trans>
-      }
-    >
-      <Icon component={SafeIcon} />
-    </Tooltip>
-  )
+  function DefaultBadge() {
+    return (
+      <Tooltip
+        placement="bottom"
+        title={
+          <Trans>
+            This project is owned by a Safe.{' '}
+            <Link href={href}>View transactions</Link>.
+          </Trans>
+        }
+      >
+        <Icon component={SafeIcon} />
+      </Tooltip>
+    )
+  }
+
+  // Red dot that appears conditionally next to Safe badge when connected user is on multisig and has unsiged tranctions
+  function Notice() {
+    return (
+      <div
+        style={{
+          position: 'absolute',
+          backgroundColor: 'red',
+          borderRadius: '100%',
+          width: '6px',
+          height: '6px',
+          top: '0px',
+          left: '13px',
+        }}
+      />
+    )
+  }
+
+  // This shows a notice of queued transaction count to users with connected wallets on the multisig
+  // Using this component so `useQueuedSafeTransactions` is only called when the connected wallet is on the multisig
+  function BadgeMightHaveNotice() {
+    const { data: queuedSafeTransactions, isLoading } =
+      useQueuedSafeTransactions({
+        safeAddress: safe.address,
+      })
+    if (isLoading || !userAddress) return <DefaultBadge />
+
+    const unsignedNoncesOfUser = getUniqueNonces(
+      getUnsignedTxsOfUser({
+        address: userAddress,
+        transactions: queuedSafeTransactions,
+      }),
+    )
+    if (queuedSafeTransactions && !unsignedNoncesOfUser.length) {
+      return <DefaultBadge />
+    }
+    return (
+      <Tooltip
+        placement="bottom"
+        overlayStyle={{ width: 700 }}
+        title={
+          <>
+            <div>
+              <Trans>
+                {plural(unsignedNoncesOfUser.length, {
+                  one: '# transaction needs',
+                  other: '# transactions need',
+                })}{' '}
```

Comment:
**praise** nice job here!

Remember that we want to translate full sentences though. I think we can add `your signature` to the end of each sentence here, and remove the `<Trans>` tag.